app/queries/find_never_preserved_child_ids.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+class FindNeverPreservedChildIds
+  def self.queries
+    [:find_never_preserved_child_ids]
+  end
+
+  attr_reader :query_service
+  delegate :orm_class, to: :resource_factory
+  delegate :adapter, to: :query_service
+  delegate :connection, to: :adapter
+  delegate :resource_factory, to: :query_service
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def find_never_preserved_child_ids(resource:)
+    # Get IDs of resources that have preservation objects.
+    file_set_ids = adapter.connection[preservation_object_query, id: resource.id.to_s].to_a.map { |x| x[:file_set_id] }
+    # Subtract those from member_ids
+    resource.member_ids.select do |member_id|
```

Comment:
suggestion: might be clearer to use `reject` rather than `select` with negative `include?`